swatch-contracts/src/test/java/com/redhat/swatch/contract/service/export/SubscriptionCsvDataMapperServiceTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.contract.service.export;
+
+import static com.redhat.swatch.contract.repository.ReportCategory.HYPERVISOR;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.redhat.swatch.common.model.ServiceLevel;
+import com.redhat.swatch.common.model.Usage;
+import com.redhat.swatch.configuration.registry.MetricId;
+import com.redhat.swatch.contract.model.SubscriptionsExportCsvItem;
+import com.redhat.swatch.contract.repository.BillingProvider;
+import com.redhat.swatch.contract.repository.SubscriptionCapacityView;
+import com.redhat.swatch.contract.repository.SubscriptionCapacityViewMetric;
+import io.quarkus.test.junit.QuarkusTest;
+import jakarta.inject.Inject;
+import java.time.OffsetDateTime;
+import java.util.HashSet;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+public class SubscriptionCsvDataMapperServiceTest {
```

Comment:
No need to add a new test for covering this. Instead, update the existing test SubscriptionDataExporterServiceTest to verify the changes.